src/hb-ot-color-sbix-table.hh

```diff
@@ -133,7 +133,7 @@ struct SBIXStrike
     return hb_blob_create_sub_blob (sbix_blob, glyph_offset, glyph_length);
   }
 
-  bool subset (hb_subset_context_t *c) const
+  bool subset (hb_subset_context_t *c, unsigned int sbix_len, unsigned int strike_offset) const
```

Comment:
In this method you only ever use the value sbix_len - strike_offset. So I recommend that you just pass one param which is that value. Could be called something like 'available_len'

Reply:
Done.